deps/aws-sdk-cpp-1.11.283/crt/aws-crt-cpp/crt/aws-c-common/source/json.c

```diff
@@ -376,6 +376,7 @@ void aws_json_module_init(struct aws_allocator *allocator) {
 void aws_json_module_cleanup(void) {
     if (s_aws_json_module_initialized) {
         s_aws_json_module_allocator = NULL;
+        cJSON_InitHooks(NULL);
```

Comment:
can we mark our customizations in dependencies to remember to add it also when we will upgrade them in the future?

Reply:
good point. I think we can remove version number from the folder name (as what I've done for curl and uuid) so we can review and keep the custom changes during update. Create issue for that and will do per dependency before next update. teamwork issue 1051.